packages/types/src/plan.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Tracking plan
+ * @property {string} [plan.branch] The tracking plan branch name e.g. "main"
```

Comment:
For TS you don't need to set the types or call out optionality. Instead you can put the comments directly above the fields:
https://stackoverflow.com/a/51641446/5046784

Reply:
Oh, I see, let me adjust this